R/expr-categorical.R

```diff
@@ -15,7 +15,56 @@ namespace_expr_cat <- function(x) {
   self
 }
 
+#' Get the categories stored in this data type
+#'
+#' @inherit as_polars_expr return
+#' @examples
+#' df <- pl$DataFrame(
+#'   cats = factor(c("z", "z", "k", "a", "b")),
+#'   vals = factor(c(3, 1, 2, 2, 3))
+#' )
+#' df
+#'
+#' df$select(
+#'   pl$col("cats")$cat$get_categories()
+#' )
+#' df$select(
+#'   pl$col("vals")$cat$get_categories()
+#' )
 expr_cat_get_categories <- function() {
   self$`_rexpr`$cat_get_categories() |>
     wrap()
 }
+
+#' Set Ordering
+#'
+#' Determine how this categorical series should be sorted.
```

Comment:
Do you also need to update the documentation, such as adding the deprecated lifecycle badge?

Reply:
I don't think that's necessary since it will be removed in the next release. I just want to mark it deprecated to not forget to remove it after the rewrite is done but before the next release.